deebot_client/commands/xml/pos.py

```diff
@@ -0,0 +1,53 @@
+"""Error commands."""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from deebot_client.events import Position, PositionType, PositionsEvent
+from deebot_client.message import HandlingResult
+
+from .common import XmlCommandWithMessageHandling
+
+if TYPE_CHECKING:
+    from xml.etree.ElementTree import Element
+
+    from deebot_client.event_bus import EventBus
+
+
+class GetPos(XmlCommandWithMessageHandling):
+    """Get pos command."""
+
+    name = "GetPos"
+
+
+    @classmethod
+    def _handle_xml(cls, event_bus: EventBus, xml: Element) -> HandlingResult:
+        """Handle xml message and notify the correct event subscribers.
+
+        :return: A message response
+        """
+        positions = []
+
+        ret = str(ret) if (ret := xml.attrib["ret"]) else ""
+        if ret != "ok":
```

Comment:
```suggestion
        if xml.attrib["ret"] != "ok":
```

Why is `str(ret)` required?